File: Ad-Hoc/pagina18/2408/2408.py
pontuacoes = [int(x) for x in input().split()]

tam = len(pontuacoes)

for i in range(tam):
    for j in range(tam - i - 1):
        if pontuacoes[j] > pontuacoes[j + 1]:
            temp = pontuacoes[j]
            pontuacoes[j] = pontuacoes[j + 1]
            pontuacoes[j + 1] = temp

print(pontuacoes[1])

File: Ad-Hoc/pagina18/2408/2408.js
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf8").split(" ").map(Number)

const quickSort = (arr) => {
    if(arr.length <= 1) return arr;
    
    let pivo = arr[Math.floor(arr.length / 2)]
    let left = []
    let right = []
    let middle = []
    for (const value of arr) {
        if (value < pivo) left.push(value);
        if (value > pivo) right.push(value);
        if (value == pivo) middle.push(value);
    }
    
    return [...quickSort(left), ...middle, ...quickSort(right)]
}

const pontuacoes = quickSort(input)
console.log(pontuacoes[1])